fabtests/functional/rdm_tagged_peek.c

```diff
@@ -44,26 +44,8 @@
 
 static struct fi_context fi_context;
 
-static int wait_for_send_comp(int count)
-{
-	int ret, completions = 0;
-	struct fi_cq_tagged_entry comp;
-
-	do {
-		ret = fi_cq_sread(txcq, &comp, 1, NULL, -1);
-		if (ret != 1) {
-			FT_PRINTERR("fi_cq_sread", ret);
-			return ret;
-		}
-		completions++;
-	} while (completions < count);
-
-	return 0;
-}
```

Comment:
Why did you remove this function, rather than just changing it, so it doesn't call fi_cq_sread?

Reply:
I thought it does the same thing with `ft_get_tx_comp` for sread. Why can't we replace it with the function in the common code?